typescript/sdk/src/consts/multisigIsmVerifyCosts.ts

```diff
@@ -0,0 +1,210 @@
+/* eslint-disable sort-keys */
```

Comment:
I just converted the file above to ts.
We should avoid imports of files other than js/ts in the SDK because it weakens our compatibility with other JS tooling. For example, by now removing all the json imports, `ts-node`won't complain about the `assert {type json` warning anymore

Reply:
Love this tidbit. TFTI(nfo)

Reply:
@yorhodes yes that's the tradeoff but the process of converting is just prepending 
`export const multisigIsmVerifyCosts = `. The benefits feel worth it to me. I'll modify the eslintignore config so we don't need that eslint disable comment at the top.